package.json

```
@@ -2,8 +2,7 @@
   "name": "@exodus/hashgraph-sdk",
   "version": "1.3.0-exodus.21",
   "description": "Hedera™ Hashgraph SDK for JavaScript/TypeScript",
-  "main": "./lib/index-node.js",
-  "browser": "./lib/index-web.js",
+  "main": "./lib/index-web.js",
```

Let's try this instead:

```suggestion
  "main": "./lib/index-node.js",
  "browser": "./lib/index-web.js",
  "exports": {
    "./index-node": "./lib/index-node.js",
    "./index-web": "./lib/index-web.js",
    ".": {
      "browser": "./lib/index-web.js",
      "node": "./lib/index-node.js",
      "default": "./lib/index-web.js"
    }
  },
```

---

This way, webpack should resolve to `index-web` in browser contexts.
Also, importing `@exodus/hashgraph-sdk/index-web` should resolve to `index-web`

---

Doesn't seem to work with webpack 4. Let's just go with your pr ).